dspace/config/spring/api/subscriptions_email_configuration.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    The contents of this file are subject to the license and copyright
+    detailed in the LICENSE and NOTICE files at the root of the source
+    tree and available online at
+
+    http://www.dspace.org/license/
+
+-->
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:util="http://www.springframework.org/schema/util"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+           http://www.springframework.org/schema/beans/spring-beans-2.5.xsd
+           http://www.springframework.org/schema/util
+           http://www.springframework.org/schema/util/spring-util.xsd">
+
+    <bean class="org.dspace.subscriptions.SubscriptionEmailNotificationServiceImpl">
+        <constructor-arg name="subscriptionType2generators">
+            <map>
+                <entry key="content" value-ref="contentNotifyGenerator"/>
+            </map>
+        </constructor-arg>
+        <constructor-arg name="contentUpdates">
+            <map>
+                <entry key="community" value-ref="communityUpdates"/>
+                <entry key="collection" value-ref="collectionUpdates"/>
+                <entry key="item" value-ref="itemUpdates"/>
+            </map>
+        </constructor-arg>
+        <constructor-arg name="param2values">
+            <map>
+                <entry key="frequency" value-ref="frequencyValues"/>
```

Comment:
Same here, this "frequency" setting is REQUIRED.  It cannot be changed as it's hardcoded into the Java Code in this PR.  We seem to be implying this is configurable when it is not.

It's also not currently possible to add new `<entry>` to this map as they will just be ignored.